.gitignore

```diff
@@ -59,6 +68,8 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+.skbuild-info.json
+uv.lock
```

Comment:
I think its better for reproducibility to remove uv.lock from .gitignore and include the uv.lock file in this PR